byceps/blueprints/admin/gallery/forms.py

```diff
@@ -18,7 +18,7 @@
 class GalleryCreateForm(LocalizedForm):
     slug = StringField(lazy_gettext('Slug'), [InputRequired()])
     title = StringField(lazy_gettext('Title'), [InputRequired()])
-    hidden = BooleanField(lazy_gettext('hidden'), [InputRequired()])
+    hidden = BooleanField(lazy_gettext('hidden'), [])
```

Comment:
The list that makes up the second parameter can be removed altogether.

Reply:
Thanks for noticing this. I will get it fixed.